tests/components/roborock/snapshots/test_vacuum.ambr

```diff
@@ -0,0 +1,24 @@
+# serializer version: 1
+# name: test_get_maps
+  dict({
+    'vacuum.roborock_s7_maxv': dict({
+      'maps': list([
```

Comment:
Somewhere it says there can only be returned the active map.
So should this really be a list or not just return the correct dict for the current map?

Reply:
It caches all of the maps that is has seen - and it should see them all on startup.